web/libs/datamanager/src/components/MainView/GridView/GridView.js

```diff
@@ -25,15 +25,35 @@ const GridHeader = observer(({ row, selected }) => {
 
 const GridBody = observer(({ row, fields }) => {
   const dataFields = fields.filter((f) => f.parent?.alias === "data");
+  const thumbnail = dataFields.filter(field=>field.alias=="thumbnail")
```

Comment:
Please, don't mix two different ideas in the same PR. Let's split it to another PR.

Reply:
The fix is done.
I created : https://github.com/HumanSignal/label-studio/pull/5769